Detectors/TPC/calibration/include/TPCCalibration/CalibdEdx.h

```diff
@@ -0,0 +1,108 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_TPC_CALIBDEDX_H_
+#define ALICEO2_TPC_CALIBDEDX_H_
+
+#include <string>
+#include <array>
+
+// o2 includes
+#include "DataFormatsTPC/TrackTPC.h"
+#include "DataFormatsTPC/dEdxInfo.h"
+#include "CCDB/CcdbObjectInfo.h"
+#include "DetectorsCalibration/TimeSlotCalibration.h"
+#include "DetectorsCalibration/TimeSlot.h"
+#include "TPCCalibration/FastHisto.h"
+
+namespace o2
+{
+namespace tpc
+{
```

Comment:
please change this everywhere
```suggestion
namespace o2::tpc
{
```